src/Microsoft.DocAsCode.Plugins/IFileLinkInfo.cs

```diff
@@ -46,8 +46,8 @@ public interface IFileLinkInfo
         bool IsResolved { get; }
 
         /// <summary>
-        /// The group that current link belongs to
+        /// The group information that current link belongs to
         /// </summary>
-        string Group { get; }
```

Comment:
breaking change?

Reply:
no worry, I'll make the change with it at the same time.